internal/datastore/postgres/migrations/driver.go

```diff
@@ -37,6 +38,13 @@ func NewAlembicPostgresDriver(ctx context.Context, url string) (*AlembicPostgres
 	pgxcommon.ConfigurePGXLogger(connConfig)
 	pgxcommon.ConfigureOTELTracer(connConfig)
 
+	if credentialsProvider != nil {
```

Comment:
We should log that the credentials provider is being used and what kind it is

Reply:
added